src/components/FilterBarPopup/FilterBarPopup.stories.tsx

```diff
@@ -0,0 +1,426 @@
+  {
+    id: "boroughs",
+    name: "Boroughs",
+    items: [
+      { id: "bronx", name: "Bronx" },
+      { id: "manhattan", name: "Manhattan" },
+      { id: "staten-island", name: "Staten Island" },
+    ],
+  },
+  {
+    id: "accessibility",
+    name: "Accessibility",
+    items: [
+      { id: "fully", name: "Fully accessible" },
+      { id: "partially", name: "Partially accessible" },
+      { id: "not", name: "Not accessible" },
+    ],
+  },
+  {
+    id: "subjects",
+    name: "Subjects",
+    items: [
+      {
+        id: "art",
+        name: "Art",
+        children: [
+          { id: "architecture", name: "Architecture" },
+          { id: "art-history", name: "Art History" },
+          { id: "artists-books", name: "Artists' Books" },
+          { id: "photography", name: "Photography" },
+          { id: "prints", name: "Prints" },
+        ],
+      },
+      { id: "genealogy", name: "Genealogy" },
+      { id: "government", name: "Government" },
+      {
+        id: "history",
+        name: "History",
+        children: [
+          { id: "african-american", name: "African American History" },
+          { id: "united-states", name: "United States History" },
+          { id: "world", name: "World History" },
+        ],
+      },
+    ],
+  },
+];
+const shortMultiSelectItems = [...multiSelectItems];
+shortMultiSelectItems.pop();
+
+const meta: Meta<typeof FilterBarPopup> = {
+  title: "Components/Form Elements/FilterBarPopup",
+  component: FilterBarPopup,
+  argTypes: {
+    className: { control: false },
+    heading: { control: false },
+    id: { control: false },
+    isOpen: { table: { disable: true } },
+    onClear: {
+      description:
+        "Function to clear all selectedItems - needs to be provided by consuming app",
+    },
+    onClose: { table: { disable: true } },
+    onOpen: { table: { disable: true } },
+    onSubmit: {
+      description:
+        "Function to handle submit of all selectedItems - needs to be provided by consuming app",
+    },
+    selectedItems: { control: false },
+    totalResults: { control: false },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof FilterBarPopup>;
+
+/**
+ * Main Story for the FilterBarPopup component. This must contains the `args`
+ * and `parameters` properties in this object.
+ */
+export const WithControls: Story = {
+  args: {
+    id: "filterbar-popup-id",
+    heading: "FilterBarPopup",
+  },
+  render: (args) => <FilterBarPopupStory {...args} />,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=87515-97592",
+    },
+    jest: ["FilterBarPopup.test.tsx"],
+  },
+};
+
+// The following are additional FilterBarPopup example Stories.
+export const FilterBarPopupHeading: Story = {
+  render: () => <FilterBarPopupHeadingStory />,
+};
+export const MultipleChildren: Story = {
+  render: () => <FilterBarChildrenStory />,
+};
+export const TagSetExample: Story = {
```

Comment:
You're not showing the `TagSetExample` in the docs.

Reply:
Replaced this with a totalResults example.